src/stringProcessing/url.js

```diff
@@ -81,6 +81,21 @@ function getHostname( url ) {
 	return url.hostname;
 }
 
+/**
+ * Returns the protocol of a URL.
```

Comment:
see terminology note about "protocol"

Reply:
Protocol now refers to the first part of the URL, before the `//`, which includes the colon.

Reply:
I did this because Node's url.parse method returns the protocol including the colon.